catalog/transformer/catalog-transformer-pptx/pom.xml

```diff
@@ -153,22 +158,26 @@
                         <Import-Package>
                             !com.ibm.icu.text,
                             !org.etsi.uri.x01903.v14.*,
-                            ddf.catalog.*,
-                            org.slf4j.*,
                             com.google.common.io.*,
-                            org.xml.sax.*,
-                            org.w3c.dom.*,
-                            javax.xml.*,
+                            ddf.catalog.*,
                             javax.imageio.*,
+                            javax.xml.*,
                             org.apache.commons.logging.*
+                            org.slf4j.*,
+                            org.w3c.dom.*,
+                            org.xml.sax.*,
                         </Import-Package>
                         <Embed-Dependency>
-                            catalog-core-api-impl,
-                            platform-util,
+	                        catalog-core-api-impl,
                             catalog-transformer-common,
-
+                            commons-io,
+                            commons-lang3,
+                            failsafe,
```

Comment:
This is the only added thing in these sections (all the other `Import-Package` and `Embed-Dependency` sections were just rearranged).

Also the Dependency on 141-145 was added.